test/graphConfig.js

```diff
@@ -2,25 +2,78 @@ var GraphConfig = require('../lib/graphConfig');
 var test = require('tape');
 var fs = require('fs');
 var path = require('path');
+var assign = require('lodash.assign');
 var properties = require('properties');
 var nconf = require('nconf');
 
```

Comment:
This test setup is big and hairy and I don't understand it after looking at it for 2 seconds. My small infant sized brain is overwhelmed.